Vec2[] boundaryPoints
NoFlyZone[] noFlyZones
Road[] roads
Vec2 startPoint
Vec2 startVelocity
Vec2[] wayPoints